package.json

```diff
@@ -37,7 +37,7 @@
   "dependencies": {
     "memoize-one": "^5.0.0",
     "react-onclickoutside": "^6.7.1",
-    "typescript-plugin-inner-jsx": "^0.1.7"
+    "typescript-plugin-inner-jsx": "^0.1.9"
```

Comment:
Even though this is definitely better it does not remove the dependency on the one who installs the lib.

Our ultimate goal should be that this dependency is no longer needed (as we deliver the transpiled sources).

Reply:
I will continue thinking about that. Thanks